gcc/rust/checks/errors/borrowck/polonius/rust-polonius.h

```diff
@@ -35,7 +35,7 @@ struct FullPoint
   bool mid;
 
   explicit FullPoint (Point point)
-    : bb (point >> 16), stmt ((point & ~(1 << 16)) >> 1), mid (point & 1)
+    : bb (point >> 16), stmt ((point >> 1) & ((1 << 15) - 1)), mid (point & 1)
```

Comment:
will reiterate that this seems very complicated - wouldn't it be worth passing around multiple integers instead of just one?

Reply:
Discussed in previous PR. No, Polonius wants one `usize`.